pkg/cli/cli.go

```diff
@@ -204,7 +204,7 @@ func (c *cli) initialize() error {
 	case len(c.cliPluginKeys) != 0:
 		// Filter plugins by keys passed in CLI.
 		filterKeys = c.cliPluginKeys
-	case c.configured && !projectConfig.IsV1():
+	case c.configured && projectConfig.IsV3():
```

Comment:
Is not missing here the `case c.configured && projectConfig.IsV2():`  so?

Reply:
No, v2 projects do not have a `layout` key.

Reply:
You can close this one as solved.